mkdocs.yml

```
@@ -13,13 +13,13 @@ extra_css:
 
 theme:
   name: material
-  logo: assets/distilabel-icon.svg
+  logo: assets/alembic.svg
```

The rename to logo mentioned before should also be changed here if you agree on applying it, otherwise feel free to discard 👍🏻 

```suggestion
  logo: assets/logo.svg
```